docs/ui/schema-conv/spanner-draft.md

```diff
@@ -82,4 +82,37 @@ Apart from the existing indexes for the source database, users can also add seco
 
 ![](https://services.google.com/fh/files/helpcenter/asset-6xbpmfyeplw.png)
 
-![](https://services.google.com/fh/files/helpcenter/asset-ck63akvjank.png)
\ No newline at end of file
+![](https://services.google.com/fh/files/helpcenter/asset-ck63akvjank.png)
+
+## Auto-Generated Columns
+*Only Supported for source database MySQL*
+
+Auto-Generated Columns populate Spanner columns automatically if no value is provided. Currently Spanner Migration Tool support the following techniques for auto-generation:
+1. UUID function: Generate a UUID (v4) as part of a table’s primary key DEFAULT expression.
+2. Bit reverse function: Map existing integer keys using the same logic as a bit-reversed sequence to avoid hotspotting.
+
+### UUID
+
+The default recommendation for primary keys in Spanner is to use a Universally Unique Identifier, or UUID. Users can convert existing columns to be filled by UUID by choosing the **Edit** option in a table and under the **Auto-Generated** column choosing UUID.
```

Comment:
please add links to the corresponding feature description on the public spanner docs.